src/Primitive/Listt.php

```diff
@@ -125,6 +129,8 @@ public function getEmpty()
 
     /**
      * @inheritdoc
+     *
+     * @throws TypeMismatchError
```

Comment:
Why you need this exception here?

Reply:
Because it is thrown in `concat` and not indicated in the docblock.

Since PHP does not have declared exceptions, indicating the exception with an annotation is good practice.